probe/kubernetes/client.go

```diff
@@ -170,47 +166,75 @@ func (c *client) isResourceSupported(groupVersion schema.GroupVersion, resource
 	return false, nil
 }
 
-func (c *client) setupStore(kclient rest.Interface, resource string, itemType interface{}, nonDefaultStore cache.Store) cache.Store {
-	lw := cache.NewListWatchFromClient(kclient, resource, metav1.NamespaceAll, fields.Everything())
+func (c *client) setupStore(resource string, nonDefaultStore cache.Store) cache.Store {
```

Comment:
Now that this function does so little, we can ditch the last arg and change the one non-nil call site to directly call `c.runReflectorUntil` instead.